python/ray/dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -63,6 +63,22 @@ def max_plus_pending(max_resource, pending_resource):
             ),
         ],
     ),
+    Panel(
```

Comment:
let's make this graph (and the other tasks graphs) not "Stacked". I think the "stacked" graphs can be ugly and hard to compare relative values.

When not stacked, we may also want to make the graphs not "filled" either so it's easier to see each line.

Reply:
done